collectors/python.d.plugin/sslcheck/sslcheck.chart.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+# Description: simple ssl expiration check netdata python.d module
+# Original Author: ccremer (github.com/ccremer)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import ssl
+import socket
+
+import datetime
+
+#try:
+#    from time import monotonic as time
+#except ImportError:
+#    from time import time
+
+from bases.FrameworkServices.SimpleService import SimpleService
+
+
+DAYS_UNTIL_EXPIRATION = 'daysuntilexpiration'
+
+ORDER = ['daysuntilexpiration']
+
+CHARTS = {
+    'daysuntilexpiration': {
+        'options': [None, 'days until certificate expiration', 'days', 'days until expiration', 'sslcheck.daysuntilexpiration', 'line'],
+        'lines': [
+            [DAYS_UNTIL_EXPIRATION, 'days until expiration', 'absolute', 10, 0]
```

Comment:
i suggest to remove `mul` and `div`, do mul/div in `get_data` if needed. And how about to return number of seconds? I think this will be better